setup.py

```diff
@@ -19,15 +19,15 @@
     zip_safe=False,
     install_requires=[
         'attrs>=21.2',
-        'clld>=9',
+        'clld>=9.2.2',
```

Comment:
If it runs with clld 10, I'd require that here: `clld>=10.0.0`. No need to make this backwards compatible.

Reply:
yes - I've to fix a conflict - I've forgotten to make master/pull